stubs/invoke/invoke/context.pyi

```diff
@@ -13,7 +14,7 @@ class Context(DataProxy):
     def prefix(self, command: str) -> AbstractContextManager[None]: ...
     @property
     def cwd(self) -> str: ...
-    def cd(self, path: str) -> AbstractContextManager[None]: ...
+    def cd(self, path: PathLike | str) -> AbstractContextManager[None]: ...
```

Comment:
We have an alias in `_typeshed for this union, it would be good to use it here:

https://github.com/python/typeshed/blob/6b98f7307ba12b86a44a8499bab58ae87e768529/stdlib/_typeshed/__init__.pyi#L134

```suggestion
    def cd(self, path: StrPath) -> AbstractContextManager[None]: ...
```

Reply:
Nice! Fixed.